entries/p/pixel.labcorp.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Pixel by Lapcorp": {
+    "domain": "pixel.labcorp.com",
+    "url": "https://www.pixel.labcorp.com",
+    "img": "pixel.labcorp.com.png",
+    "tfa": [
+      "totp"
+    ],
```

Comment:
Their [documentation / support site](https://support.pixel.labcorp.com/hc/en-us) doesn't have any information about two-factor auth, unfortunately.

Reply:
Since there is no documentation available, please could you provide us with screenshots of the setup/login process as evidence of 2FA? Please remember to block out any personal information.

Reply:
Sure -

<img width="731" alt="Screen_Shot_2021-08-01_at_1_23_48_PM" src="https://user-images.githubusercontent.com/630449/127783017-d632d42c-425f-4cf9-9c3f-6b359ee24e57.png">

<img width="1124" alt="Screen Shot 2021-08-01 at 1 24 43 PM" src="https://user-images.githubusercontent.com/630449/127783040-49e55dd0-b171-458f-a68c-ce09d35b3432.png">